neovim/plugin/decorators.py

```diff
@@ -141,14 +155,23 @@ def dec(f):
 
         f._nvim_rpc_spec = {
             'type': 'function',
-            'name': name,
+            'name': function_name,
             'sync': rpc_sync,
             'opts': opts
         }
         return f
     return dec
 
 
+def capitalize_name(name):
```

Comment:
I do not like this kind of magic. For plugins that use this, it would be better to break python conventions and name the implementation function literally what the vimL side name is. Also, `_` is not forbidden in vimL.

Reply:
Well, a matter of style I think. I would no like to have a function/command named `My_function_something`, and you can still pass a name if people want that name.